src/private/interfaces.ts

```diff
@@ -93,6 +94,11 @@ export interface FilePreviewParameters {
    * This field should be used to restore to a specific state within an entity, such as scrolling to or activating a specific piece of content.
    */
   subEntityId?: string;
+
+  /**
+   * Optional; indicates the mode in which file should be opened. Takes precedence over edit mode.
+   */
+  viewerAction?: 'view' | 'edit' | 'editNew';
```

Comment:
Can you please add a new enum for these options - view, edit, editNew?

Reply:
Have added the enum.